src/tools/BrushTool.js

```diff
@@ -50,17 +45,16 @@ export default class BrushTool extends BaseBrushTool {
     const { rows, columns } = eventData.image;
     const { x, y } = mousePosition;
 
-    if (x < 0 || x > columns ||
-      y < 0 || y > rows) {
+    if (x < 0 || x > columns || y < 0 || y > rows) {
```

Comment:
I don't have strong feelings about this either way, but I put this on two lines just because I think the triple `||` is a little confusing. Eh ¯\\_(ツ)\_/¯.

Reply:
That's my auto-formatter being too opinionated. I think `prettier` is just enforcing a line length here.

Reply:
> Prettier is an opinionated code formatter.

Reply:
But yeah, its not really an issue 👍 .